doc/doxygen/headers/preconditioners.h

```diff
@@ -113,37 +108,28 @@
  *
  * <h4>Application of the preconditioner</h4>
  *
- * Preconditioners in deal.II are just considered linear operators.
- * Therefore, in order to be used by deal.II solvers, preconditioners must
- * conform to the standard matrix interface, namely the functions
- * @code
- * void  vmult (VECTOR& dst, const VECTOR& src) const;
- * void  Tvmult (VECTOR& dst, const VECTOR& src) const;
- * @endcode
- * These functions apply the preconditioning operator to the source
- * vector $src$ and return the result in $dst$ as $dst=P^{-1}src$ or
- * $dst=P^{-T}src$. Preconditioned iterative
- * dolvers use these <tt>vmult()</tt> functions of the preconditioner.
- * Some solvers may also use <tt>Tvmult()</tt>.
+ * Preconditioners in deal.II have the same interface as matrices. Therefore,
+ * in order to be used by deal.II solvers, preconditioners must conform to the
+ * standard matrix interface; see the discussion in @ref Concepts for more
+ * information on type requirements.
  *
  * <h4>Relaxation methods</h4>
  *
- * Additional to the interface described above, some preconditioners
- * like SOR and Jacobi have been known as iterative methods
- * themselves. For them, an additional interface exists, consisting of
- * the functions
+ * Additional to the interface described above, some preconditioners like SOR
+ * and Jacobi have been known as iterative methods themselves. For them, an
```

Comment:
Again not your fault, but this is historically not correct -- these were all methods that were used as iterative solvers decades or centuries before people starting to think of them as preconditioners (which at the time was not a known concept). Maybe write it as
```
  Some preconditioners, like SOR and Jacobi, were used as iterative solvers long before they were used
  as preconditioners. Thus, they also satisfy the requirements of solvers and smoothers by defining the following,
  additional, interface:
```